### PR TITLE
<BarChart /> - align text in a tooltip to the start of the line

### PR DESCRIPTION
**Pull request by maxvektor**
### 🔦 Summary
I our layout at zeplin  text in a popup starts at the start of the line. This is a fix.
...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

File: src/BarChart/BarChart.js [hunk 1]
```diff
@@ -135,7 +136,7 @@ class BarChart extends React.PureComponent {
   };
 
   render() {
-    const { dataHook, items, deprecatedColors, total } = this.props;
```
Comment (maxvektor) [inline src/BarChart/BarChart.js]: just unused value

**Comment by argshook** [comment]
in wsr zeplin i see that it's set in the center.
@milkyfruit pretty please, should BarChart tooltip be aligned to center or to the left?

also applies to https://github.com/wix/wix-style-react/pull/4506

**Comment by milkyfruit** [comment]
@argshook  link to specs: https://zpl.io/a8RKzmJ
Tooltip's default `placement` must be `top`.
Tooltip text must be aligned left.

**Comment by argshook** [comment]
just fyi: something's wonky with visual tests, once green i'll merge

**Pull request merged**